tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/cell_data.py

```diff
@@ -27,17 +32,44 @@ def carrier_name(self):
         Wait(self.marionette).until(expected.element_displayed(element))
         return element.text
 
+    @property
+    def screen_element(self):
+        return self.marionette.find_element(*self._page_locator)
+
+    # workaround for elements within CellDataPrompt going stale
```

Comment:
You mean that is due to https://bugzilla.mozilla.org/show_bug.cgi?id=1202246 , right?
Could you mention that bug in this comment?